roles/openshift_health_checker/test/docker_storage_test.py

```diff
@@ -56,14 +57,35 @@ def test_is_active(is_containerized, group_names, is_active):
     (
         dict(info={
             "Driver": "overlay2",
-            "DriverStatus": []
+            "DriverStatus": [("Backing Filesystem", "xfs")],
         }),
         False,
         [],
     ),
     (
         dict(info={
             "Driver": "overlay",
+            "DriverStatus": [("Backing Filesystem", "btrfs")],
+        }),
+        True,
+        ["storage is type 'btrfs'", "only supported with\n'xfs'"],
```

Comment:
Yeah, it does, which is actually why it wasn't there before. But the `\n` doesn't truly matter, it's the generic problem of having error string outputs instead of defined exceptions. I'd like to refactor as much as possible into exceptions at some point. It's tech debt for now.